capsules/detector_safety_gear_openvino/meta.conf

```
@@ -0,0 +1,2 @@
+[about]
+api_compatibility_version = 0.2
```

```suggestion
api_compatibility_version = 0.3
```

The change from futures to queues requires a bump to 0.3